src/main/resources/resources/logisim/strings/gui/gui_pl.properties

```diff
@@ -746,7 +745,7 @@ ttyNeedsFileError = Użycie "--tty" wymaga podania nazwy pliku w wierszu polece
 # start/TtyInterface.java
 #
 loadIoError = Błąd podczas odczytu pliku obrazu
-loadNoRamError = Nie znaleziono pamięci RAM dla opcji "--load".
+loadNoRamError = Nie znaleziono pamięci MEM dla opcji "--load".
```

Comment:
Same doubts about `MEM` here. Or, in fact , I have no doubts. I actually know this makes no sense in Polish.